resources/zib/zib-FamilySituationChild.xml

```diff
@@ -0,0 +1,296 @@
+    </element>
+    <element id="List.mode">
+      <path value="List.mode"/>
+      <comment value="This element has no functional counterpart in the zib and cannot be implicitly mapped to other concepts. Unless a more appropriate status is recorded, it is assumed that only _working_ or _snapshot_ List resources will be exchanged, depending on the functioning of the source system."/>
+    </element>
+    <element id="List.code">
+      <path value="List.code"/>
+      <min value="1"/>
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct"/>
+          <code value="55801000146105"/>
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="List.subject">
+      <path value="List.subject"/>
+      <type>
+        <code value="Reference"/>
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Patient"/>
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Group"/>
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Device"/>
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Location"/>
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-Patient"/>
+      </type>
+    </element>
+    <element id="List.note">
+      <path value="List.note"/>
+      <max value="1"/>
+    </element>
+    <element id="List.note.text">
+      <path value="List.note.text"/>
+      <short value="Comment"/>
+      <definition value="A comment on the childs family situation."/>
+      <alias value="Toelichting"/>
+      <mapping>
+        <identity value="zib-familysituationchild-v1.2-2020EN"/>
+        <map value="NL-CM:7.16.11"/>
+        <comment value="Comment"/>
+      </mapping>
+    </element>
+    <element id="List.entry">
+      <path value="List.entry"/>
+      <slicing>
+        <discriminator>
+          <type value="profile"/>
+          <path value="item.resolve()"/>
+        </discriminator>
+        <discriminator>
+          <type value="value"/>
+          <path value="extension('http://nictiz.nl/fhir/StructureDefinition/ext-List-EntryCategory').value"/>
+        </discriminator>
+        <rules value="open"/>
+      </slicing>
+    </element>
+    <element id="List.entry.extension:category">
+      <path value="List.entry.extension"/>
+      <sliceName value="category"/>
+      <type>
+        <code value="Extension"/>
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-List-EntryCategory"/>
+      </type>
+      <isModifier value="false"/>
+    </element>
+    <element id="List.entry:familyComposition">
+      <path value="List.entry"/>
+      <sliceName value="familyComposition"/>
+    </element>
+    <element id="List.entry:familyComposition.extension:category">
+      <path value="List.entry.extension"/>
+      <sliceName value="category"/>
+      <max value="0"/>
```

Comment:
It feels somewhat uneasy to (explicitly) add the extension on the main element and than forbid it again in some of the slices. Wouldn't it be more straightforward to always include it (using the definition codes from the zib)? Or alternatively, to only explicitly add it in the sibling and child slices?

The first option would also mean that the discriminator only needs to look at the extension.

Reply:
I can live with the second approach.

Although the first approach uses only one discriminator I do not really agree that adding extensions that aren't really required for anything is more elegant. The slicing should just work.